hack/local-up-kubeedge.sh

```diff
@@ -46,6 +52,22 @@ function check_prerequisites {
   fi
 }
 
+function verify_crio_installed {
+  # Commands to check if cri-o is installed
+  if ! command -v crio &> /dev/null; then
+    echo "cri-o is not found"
+    exit 1
+  fi
+}
+
+function verify_containerd_installed {
```

Comment:
`verify_containerd_installed` has achieved in hack/lib/install.sh. 😄

Reply:
Resolved